ttnn/cpp/ttnn/operations/moreh/moreh_linear_backward/device/moreh_linear_backward_device_operation.cpp

```diff
@@ -0,0 +1,87 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "moreh_linear_backward_device_operation.hpp"
+
+#include <cstdint>
+
+#include "tt_dnn/op_library/moreh_helper_functions.hpp"
+#include "ttnn/operations/core/compute_kernel/compute_kernel_config.hpp"
+#include "ttnn/tensor/tensor.hpp"
+
+namespace ttnn::operations::moreh::moreh_linear_backward {
+
+void MorehBiasAddBackwardOperation::validate_inputs(
+    const operation_attributes_t& operation_attributes, const tensor_args_t& tensor_args) {
+    auto& bias_grad = tensor_args.bias_grad;
+
+    if (bias_grad.has_value()) {
+        auto bias_grad_shape = bias_grad->get_shape();
+        auto bias_grad_tensor = bias_grad.value();
+        TT_ASSERT(
```

Comment:
```suggestion
        TT_FATAL(
```

Reply:
resolved in [0e5e4b5](https://github.com/tenstorrent/tt-metal/pull/12812/commits/0e5e4b5b4fa51a995cad3d55cff760fab36b2381)